### DISCUSSION TITLE
Welcome to workflow-group-3 Discussions!

### DISCUSSION THREAD

**Discussion by galalkoro98** [HYF-Class19/workflows-group-3#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sammou00** [comment]
Good job, I really like your README.md.
